src/SFA.DAS.AdminService.Application.Interfaces/SFA.DAS.AdminService.Application.Interfaces.csproj

```diff
@@ -7,7 +7,7 @@
 
   <ItemGroup>
     <PackageReference Include="EPPlus" Version="6.0.7" />
-    <PackageReference Include="SFA.DAS.AssessorService.Shared" Version="0.1.242" />
+    <PackageReference Include="SFA.DAS.AssessorService.Shared" Version="0.1.243-prerelease-38" />
```

Comment:
@ChrisJWoodcock in the future would a 0.1.243 version of SFA.DAS.AssessorService.Shared published and Admin service be updated to that?

Reply:
When this is ready to be merged to master, first the assessor code would be merged and the nuget package published and then the master published nuget package would be used in the admin code and the admin code merged to master